<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>769</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="Plot_Widget" native="true">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>811</width>
      <height>591</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveEvent_Button">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>80</y>
      <width>99</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Event</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LoadFile_Button">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>180</y>
      <width>99</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Load File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Zoom_In_x_Button">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>660</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Zoom_Out_x_Button">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>660</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Zoom_In_y_Button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>600</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>V^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Zoom_Out_y_Button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>560</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>^V</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="x_Bar">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>640</y>
      <width>811</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="y_Bar">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>20</width>
      <height>591</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="EventFile_Label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>/usr/src/QT/Extras/qcustomplot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
